src/main/kotlin/exercise3/task2/ParenthesesClusters.kt

```diff
@@ -24,7 +24,45 @@ package exercise3.task2
  */
 
 internal fun String.splitToBracketsClusters(): List<String> {
-    TODO("Implement me!!!")
+    val opening = "({["
+    val closing = ")}]"
+    val clusterList = ArrayList<String>()
+    val emptyList = ArrayList<String>()
```

Comment:
This is only used as a return value at the end of the function. 
It's better to move it's declaration closer to the places where it's used.